### DISCUSSION TITLE
My style isn't generated and defaults to Tailwind's Preflight option

### DISCUSSION THREAD

**Discussion by Fredenck** [tailwindlabs/tailwindcss#11741] [Help] [answered]
Building my page, I see:

<img width="399" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/40157329/09984086-d742-4d57-ba18-ac0fc80c1ecf">

This is just a simply list with disc bullet points for each item. However, the generated CSS is defaulting to Tailwind's Preflight style sheet: 
<img width="550" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/40157329/d2f70139-75ba-48cf-a9d0-f2be216d23a6">
because it doesn't generate a .list-desc{ ...} css piece.

For reference, I'm using Contentlayer to parse and transform my mdx.

versions:
<img width="291" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/40157329/b3d67095-ab14-419a-b6fa-27186f2ac626">

**Comment by wongjn** [comment]
Consider checking:
- The `content` file globs sufficiently cover the file that the `list-*` classes appear in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names).

**Reply by Fredenck** [reply]
I have 
<img width="352" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/40157329/d73297d4-cf36-4ef4-ace5-55ee41d9fd0c">
where my posts/ folder is the same level as my tailwind.config.ts file
and, 
I'm using 
<img width="727" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/40157329/e1dbd13f-fea7-47b0-84ca-9673e09142f8">
for my class names so there shouldn't be string concatenation/interpolation

Another odd thing is that only _some_ classes render correctly in my mdx. For example, 
`<img src="/fill.jpg" alt="fill" className='w-1/4 rounded-lg mx-4 my-4'></img>` works, but replacing w-1/4 with w-1/3 or w-1/2 won't be valid and the image size will go to its default

**Reply by wongjn** [reply, marked as answer]
Consider checking:
- The `ul`, `ol` functions are in a file covered by the `content` file globs in the Tailwind configuration. 
- The file that contains the code that contains the `<img>` classes are covered by the `content` file globs in the Tailwind configuration.

**Reply by Fredenck** [reply]
Wow, your first bullet point was right, my tailwind.config.ts wasn't globbing that file. I never expected that because it seemed to globbed, but only for some specific classNames.

Thanks!
